src/views/Dog/Dog.js

```diff
@@ -0,0 +1,21 @@
+import { fetchDogById } from '../../services/dogs';
+import { useEffect, useState } from 'react';
+
+import DogDetail from '../../components/Dog/DogDetail';
+
+export default function Dog(props) {
+  const id = props.match.params.id;
+  const [dog, setDog] = useState([]);
```

Comment:
fetchDogById returns and object, so we might want to set the default state to an empty object as well